inference-engine/tests/functional/plugin/shared/src/behavior/config.cpp

```diff
@@ -0,0 +1,307 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+#include <tuple>
+#include <vector>
+#include <string>
+
+#include <ie_core.hpp>
+#include "ie_common.h"
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/plugin_cache.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "functional_test_utils/blob_utils.hpp"
+#include <threading/ie_executor_manager.hpp>
+#include "ngraph_functions/pass/convert_prc.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "behavior/config.hpp"
+
+
+namespace LayerTestsDefinitions {
+std::string CorrectConfigTests::getTestCaseName(testing::TestParamInfo<ConfigParams> obj) {
+    InferenceEngine::Precision  netPrecision;
+    std::string targetDevice;
+    std::map<std::string, std::string> configuration;
+    std::tie(netPrecision, targetDevice, configuration) = obj.param;
+    std::ostringstream result;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice;
+    if (!configuration.empty()) {
+        result << "configItem=" << configuration.begin()->first << "_" << configuration.begin()->second;
+    }
+    return result.str();
+}
+
+void CorrectConfigTests::SetUp() {
```

Comment:
The number of tests is equal to the number of these tests in the old test infra?

Reply:
Yes